.gitignore

```diff
@@ -9,3 +9,5 @@
 /.dockerignore
 /docker-compose.yml
 /yarn-error.log
+.env
+config/serviceAccountKey.json
```

Comment:
Add missing documentation for this please :)

Reply:
Yep added it here as a oneliner and described how to set it up in the README.md.